django_video_encoder/utils.py

```diff
@@ -0,0 +1,14 @@
+FULL_RESOLUTION = "full resolution"
+
+def format_label_to_codec_width_height(format_label):
+    video_codec, resolution = format_label[:-1].rsplit(" (", 1)
+    if resolution == FULL_RESOLUTION:
+        width = height = None
+    else:
+        width, height = resolution.split("x")
+    return video_codec, width, height
+
+
+def codec_width_height_to_format_label(video_codec, width, height):
+    resolution = f"{width}x{height}" if width or height else FULL_RESOLUTION
```

Comment:
This will actually create (Nonex720) or (1920xNone). The missing part will be scaled to preserve the ratio.

I just pushed a more explicit version.

Also I removed the reverse engineer util, as it was not used anymore and won't probably in the future...